.gitignore

```diff
@@ -13,4 +13,7 @@
 
 # Dependencies and coverage
 *.coverprofile
-vendor
\ No newline at end of file
+vendor
+
+# Test database
+fixtures/db
```

Comment:
For alpha development purposes, the database is just being stored in fixtures. We'll have to make the configuration more robust when we prepare for deployment.

Reply:
👍  this is akin to what you were imagining for baleen in your comment [here](https://github.com/kansaslabs/baleen/issues/8#issuecomment-541635745), right?

Reply:
Yes, this could also work for development purposes if you wanted to store the Baleen manifest on a local disk instead of S3. At some point, both here and in Baleen we'll need to have a default database location, e.g. `/data/db` if we're attaching an EBS volume to an EC2 instance or `~/.kansaslabs/rossbydb` if we're running things locally on our own computer.